3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/package.scala

```diff
@@ -155,4 +156,10 @@ package object common {
    * It has [[Monad]] instance unlike [[Validation]]
    */
   type ThrowableXor[+A] = Throwable \/ A
+
+  /**
+   * Type alias for an Option-boxed
+   * Tuple3 of a file location
+   */
+  type FileTuple = Option[(String, URI, String)]
```

Comment:
This type alias doesn't provide any benefits as we can't infer what's inside from the name

Reply:
Should I use a case class then?

Reply:
yes